StoreManagementService/src/PBJ.StoreManagementService.Api/Controllers/CommentController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AutoMapper;
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+using PBJ.StoreManagementService.Business.Services.Abstract;
+using PBJ.StoreManagementService.Models.Comment;
+
+namespace PBJ.StoreManagementService.Api.Controllers
+{
+    [ApiController]
+    [Route("api/comment")]
+    public class CommentController : ControllerBase
+    {
+        private readonly ICommentService _commentService;
+        private readonly IValidator<CommentRequestModel> _validator;
+
```

Comment:
Configure fluent validator globally. You could throw necessary errors from validator config

        public CheckDeviceValidator()
        {
            RuleFor(x => x).Custom((model, _) =>
            {
                if (model.UserId.IsNullOrEmpty())
                {
                    throw new WebApiException(errorCode: ErrorCodes.INVALID_INPUTS);
                }

                if (model.DeviceId.IsNullOrEmpty())
                {
                    throw new WebApiException(errorCode: ErrorCodes.INVALID_INPUTS);
                }
            });
        }